sympy/concrete/tests/test_sums_products.py

```diff
@@ -1011,12 +1009,15 @@ def test_issue_10156():
     assert e.factor() == \
         8*y**3*Sum(x, (x, 1, 3))*Sum(x**2, (x, 1, 9))
 
-
 def test_issue_14112():
     assert Sum((-1)**n/sqrt(n), (n, 1, oo)).is_absolutely_convergent() is S.false
     assert Sum((-1)**(2*n)/n, (n, 1, oo)).is_convergent() is S.false
     assert Sum((-2)**n + (-3)**n, (n, 1, oo)).is_convergent() is S.false
 
+def test_sin_times_absolutely_convergent():
+    assert Sum(sin(n) / n**3, (n, 1, oo)).is_convergent() is S.true
+    assert Sum(sin(n) * log(n) / n**3, (n, 1, oo)).is_convergent() is S.true
```

Comment:
Strangely, `Sum(sin(n)*log(n)/n**3, (n, 2, oo)).is_convergent()` raises NotImplementedError. The only difference is using n=2 starting index instead of 1. What gives?

Reply:
This isn't a problem with new code here, though.
```
>>> Sum(log(n)/n**3, (n, 1, oo)).is_absolutely_convergent()
True
>>> Sum(log(n)/n**3, (n, 2, oo)).is_absolutely_convergent()
NotImplementedError
```
I think this can be left as a separate issue, as it's with another method; it just came up in testing here.

Reply:
Does the failed test is related to this?

In my pc, the test `test_discrete_rv.py` works.

Reply:
The failed test is a random failure of a sampling method, which is random by nature. Another annoyance of having sampling methods. I restarted that test.

Reply:
The issue `Sum(log(n)/n**3, (n, 1, oo)).is_absolutely_convergent()` is due to the integral test. If the series starts from `2`, then `maxima` in line 514 is an empty set.

Reply:
Thanks for restarting the test.

Reply:
You are right about indentation. Would you like to correct that with this PR, or leave it for someone else?

Reply:
I can leave it for someone else. It can be labelled as `easy to fix`.
However, it seems also necessary to add a flag, because sometimes `check_interval` is not assigned and some tests fail.